src/components/PlaylistPage/LessonItem.scss

```diff
@@ -22,3 +22,7 @@
 .title {
   padding: 2px 0;
 }
+
+img {
```

Comment:
Hvis du vil style tooltip'en uten å påvirke noe annet på siden, gjør som følger.

Først, wrap tooltip'en i en div med sin egen style (kommentaren om å flytte ut `__html` gjelder fortsatt, men tok bare en copy paste fra slik det var nå):
```
<div className={styles.tooltip}>
  <div dangerouslySetInnerHTML={{__html: tooltipContent}} />
</div>
```

Deretter gjør du alle "global"-endringer inni styles.tooltip som følger:
```
.tooltip {
  img {
    max-width: 180px;
  }
}
```

(Dersom det var et "global" klassenavn satt av bootstrap du ville endre på, f.eks. klassen `.overlaytooltip`(det bare fant jeg på her), så ville du brukt `:global(.overlaytooltip)` for å unngå "mangling").